spec/fixtures/https_server.lua

```diff
@@ -0,0 +1,235 @@
+local https_server = {}
+https_server.__index = https_server
+
+
+local fmt = string.format
+local nginx_tpl_file = require 'spec.fixtures.nginx_conf_template'
+local ngx = require 'ngx'
```

Comment:
Use double-quotes instead of single quotes, everywhere. 

https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#strings